specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -8498,6 +8515,10 @@
             "name": "VirtualMachineEvictionPolicyTypes",
             "modelAsString": true
           }
+        },
+        "scheduledEventsProfile" : {
+          "$ref":"#/definitions/ScheduledEventsProfile",
+          "description": "Specifies Scheduled Event related configurations."
         }
```

Comment:
Is this a required property?

Reply:
Not required

Reply:
Great. Thanks for confirming.

---
In reply to: [309897817](https://github.com/Azure/azure-rest-api-specs/pull/6683#discussion_r309897817) [](ancestors = 309897817)